Handle comma separated allowed roles (#391)
nhost/hasura-auth: .changeset/strange-seas-whisper.md

---
'hasura-auth': patch
---

> Fix issue where hasura auth-crashes when adding allowed roles upon signin with provider

Reviewer: ```suggestion
fix: don't crash when adding allowed roles upon sign-in with a provider
```
